framework/src/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1099,6 +1107,7 @@ protected void analyze(
         if (emptyStore == null) {
             emptyStore = newAnalysis.createEmptyStore(transfer.usesSequentialSemantics());
         }
+        cacheLookupMap.clear();
```

Comment:
Should the cache also be cleared in `setRoot`? Most other maps are cleared there.